cmd/ef_tests/state/runner_v2/parser.rs

```diff
@@ -1,10 +1,29 @@
-use std::path::PathBuf;
-
 use crate::runner_v2::{
     error::RunnerError,
     types::{Test, Tests},
 };
 
+use clap::Parser;
+
+#[derive(Parser, Debug)]
+pub struct RunnerOptions {
+    /// For running tests in a specific file (could be either a directory or a .json)
+    #[arg(
+        short,
+        long,
+        value_name = "PATH",
+        value_delimiter = ',',
+        default_value = "./vectors"
+    )]
+    pub path: String,
+    /// For running tests in specific .json files. If this is not empty, "path" flag will be ignored.
+    #[arg(short, long, value_name = "JSON_FILES", value_delimiter = ',')]
+    pub json_files: Vec<String>,
```

Comment:
better to use a higher level abstraction like PathBuf, see https://github.com/lambdaclass/ethrex/pull/3770